src/mrack/utils.py

```diff
@@ -201,6 +206,51 @@ def get_ssh_key(host, meta_host, config):
     return ssh_key
 
 
+def ssh_to_host(
+    host,
+    execute=None,
+    username=None,
+    password=None,
+    ssh_key=None,
+):
+    """SSH to the selected host."""
+    my_env = os.environ.copy()
+
+    run_args = {
+        "env": my_env,
+        "shell": True,
+    }
+
+    cmd = ["ssh"]
+    cmd.extend(["-o", "'StrictHostKeyChecking=no'"])
+    cmd.extend(["-o", "'UserKnownHostsFile=/dev/null'"])
+
+    meta_host, _domain = get_host_from_metadata(global_context["metadata"], host.name)
+    username = username or get_username(host, meta_host, global_context["config"])
+    ssh_key = ssh_key or get_ssh_key(host, meta_host, global_context["config"])
+
+    if username:
+        cmd.extend(["-l", username])
+    if ssh_key:
+        cmd.extend(["-i", ssh_key])
+    psw_input = None
+    if password and not ssh_key:
+        cmd.extend(["-o", "'PasswordAuthentication'"])
+        psw_input = f"{password}\n"
+
+    cmd.append(host.ip_addr)  # Destination
+
+    if execute:
+        cmd.extend(["-C", execute])
```

Comment:
```
$ ssh [-options] destination [command]
```
`-C` "Requests compression of all data... Compression is desirable on modem lines and other slow connections, but will only slow down things on fast networks."
Should we remove this?